cmd/krel/cmd/push.go

```diff
@@ -40,18 +40,18 @@ push --bucket=kubernetes-release-$USER
                            - Do a developer push to kubernetes-release-$USER`
 
 type pushBuildOptions struct {
-	allowDup         bool
 	bucket           string
-	ci               bool
 	dockerRegistry   string
 	extraPublishFile string
-	federation       bool
 	gcsSuffix        string
-	noUpdateLatest   bool
-	privateBucket    bool
 	releaseKind      string
 	releaseType      string
 	versionSuffix    string
+	allowDup         bool
```

Comment:
This is just sorting the types, right?

Reply:
Yes the linter is `maligned`, which is currently only able to tell us that we can safe space in the struct, but not how :see_no_evil:. But sorting the members by size gives us the best memory alignment and safes some bytes on runtime. :+1:

Reply:
Cool!